### PR TITLE
2023-3.2: pin `sqlalchemy` to >=2.0.20

### LINKED ISSUE #21: 2023-3.2: new packages
TODOs:
- [ ] xpdacq v1.0.0

### PR DESCRIPTION
**Pull request by mrakitin**
Per report in NSLS2 Slack.

Closes #21

### CODE REVIEW

**Review by dmgav — APPROVED** [review]

File: envs/env-py310.yml [hunk 1]
```diff
@@ -193,7 +196,7 @@ dependencies:
   - kkcalc
   - ppmac
   - pychx >=4.1.2
-  - xpdacq >=1.1.4
+  - xpdacq ==1.0.0
```
Comment (danielballan) [inline envs/env-py310.yml:199]: Intentionally downgraded?

**Review by danielballan — APPROVED** [review]
Looks good. Just one question to be sure.

File: envs/env-py310.yml (hunk 1 above)
Comment (mrakitin) [reply envs/env-py310.yml:199]: Yes, the latest version of `xpdacq` does not work with the latest pdfstream. It was discussed with the beamline staff.

**Review by AbbyGi — APPROVED** [review]

**Comment by mrakitin** [comment]
I added one commit to match the version of `pdfstream ==0.5.2` used in the `2022-2.0-py37` conda environment (see  https://zenodo.org/records/6462525/files/2022-2.0-py37.yml for details). I'll merge the PR once the CI passes.

The `edrixs` exclusion was due to some conflicts with `numexpr` (cc @mpmdean) - we'll have to rebuild the new, cmake-based version in conda-forge, and then we'll add it back, hopefully, the conflicts will be resolved.

File: envs/env-py310.yml (hunk 1 above)
Comment (mrakitin) [reply envs/env-py310.yml:199]: Discussed it again with the BL staff, and we need a bit older version of `pdfstream` to work correctly in tandem. This was addressed via https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/24/commits/1fda94abda2ad8ee2b8cf17484ad892d2fa86fd7.

**Comment by mpmdean** [comment]
Thanks for the note @mrakitin 

What do you think about making a new release out of the current edrixs code? And leaving creating a pyproject.toml for later?

**Comment by mrakitin** [comment]
@mpmdean, let's chat later this week to plan the release.

**Pull request merged**